terraform/common/terraform.tf

```diff
@@ -5,8 +5,8 @@ provider "aws" {
 terraform {
 
   backend "s3" {
-    bucket  = "terraform-state-002"
-    key     = "tvs/terraform-common.tfstate"
+    bucket  = "530003481352-terraform-state"
+    key     = "common/common.tfstate"
```

Comment:
This could go in production as well

Reply:
Yes, good spot. Have grouped under production namespace